docs/HowTo/Monitor/Cakeshop.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Using Cakeshop with GoQuorum
+---
+
+# Cakeshop
+
+[Cakeshop](https://github.com/ConsenSys/cakeshop) is a set of tools and APIs for working with Ethereum,
+packaged as a Java web application archive (WAR) that gets you up and running quickly.
+
+Out of the box you get:
+
+* **Blockchain explorer** - View transactions, blocks, and contracts, and see historical contract state at a point in time.
+* **Contract registry** - Keep track of deployed contracts and their code, interfaces, and state.
```

Comment:
note: the contracts are registered only if those are deployed through the Solidity sandbox. if you deploy by other means (using an external Remix IDE or manually through Truffle or other) won't show up there. 

not sure if is worth mentioning this.